### PR TITLE
WebHost: check: display exception chain one layer deep

### PR DESCRIPTION
**Pull request by Berserker66**
## What is this fixing or adding? How was this tested? If this makes graphical changes, please attach screenshots.
Before:
image

After:
image

### CODE REVIEW

File: WebHostLib/check.py [hunk 1]
```diff
@@ -108,7 +108,10 @@ def roll_options(options: Dict[str, Union[dict, str]],
                         rolled_results[f"{filename}/{i + 1}"] = roll_settings(yaml_data,
                                                                               plando_options=plando_options)
             except Exception as e:
-                results[filename] = f"Failed to generate options in {filename}: {e}"
+                if e.__cause__:
+                    results[filename] = f"Failed to generate options in {filename}: {e}, {e.__cause__:}"
```
Comment (ScipioWright) [inline WebHostLib/check.py:112]:
```suggestion
                    results[filename] = f"Failed to generate options in {filename}: {e} - {e.__cause__:}"
```

Comment (Berserker66) [inline WebHostLib/check.py:112]:
```suggestion
                    results[filename] = f"Failed to generate options in {filename}: {e} - {e.__cause__}"
```

**Review by beauxq — APPROVED** [review]
I didn't test.
I just looked up the type of `__cause__` (`BaseException | None`)

**Review by LegendaryLinux — APPROVED** [review]
Tested locally, looks good.

**Pull request merged**